app/controllers/concerns/sufia/files_controller_behavior.rb

```diff
@@ -96,5 +96,17 @@ def file_set_params
         return {} if params[:file_set].blank?
         super
       end
+
+      def find_parent_by_id
```

Comment:
I might prefer this as a guard clause to make explicit there's some errant behavior you're protecting against, after which you do processing, like so:

```ruby
return if parent_id.empty?
ActiveFedora::Base.find(parent_id)
```

But I wouldn't hold the PR over that alone.